core/flyout.js

```diff
@@ -684,6 +720,44 @@ Blockly.Flyout.prototype.layoutBlocks_ = function(blocks, gaps) {
 };
 
 /**
+ * Create and place a checkbox corresponding to the given block.
+ * @param {!Blockly.Block} block The block to associate the checkbox to.
+ * @param {number} cursorX The x position of the cursor during this layout pass.
+ * @param {number} cursorY The y position of the cursor during this layout pass.
+ * @param {!{height: number, width: number}} blockHW The height and width of the
+ *     block.
+ * @param {Element} root The svg root of the block.
+ * @private
+ */
+Blockly.Flyout.prototype.createCheckbox_ =
+    function(block, cursorX, cursorY, blockHW, root) {
+  /* eslint-disable indent */
+  var extraSpace = this.CHECKBOX_SIZE + this.CHECKBOX_MARGIN;
+  var checkboxRect = Blockly.createSvgElement('rect',
+    {
+      'class': 'blocklyFlyoutCheckbox',
+      'height': this.CHECKBOX_SIZE,
+      'width': this.CHECKBOX_SIZE,
+      'x': this.RTL ? cursorX + blockHW.width + extraSpace :
```

Comment:
Hmmm, something seems odd in RTL:
<img width="169" alt="screen shot 2016-06-24 at 11 35 57 am" src="https://cloud.githubusercontent.com/assets/120403/16342418/09055f0c-3a00-11e6-8a1e-5eb2661d9b9b.png">
Thoughts?